packages/libatasmart.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Libatasmart < Package
+  description 'LIBATASMART ATA S.M.A.R.T. Reading and Parsing Library'
+  homepage 'http://git.0pointer.net/libatasmart.git/'
+  version '0.19'    
+  license 'LGPL-2.1+'
+  compatibility 'all'
+  source_url 'http://git.0pointer.net/libatasmart.git/'
+  git_hashtag "v#{version}"
+  git_branch 'master'
```

Comment:
I don't think you need the git branch if a git tag is provided?

Reply:
Needed for broken older git servers as a workaround for `fatal: dumb http transport does not support shallow capabilities` issue.